sched/scheduler/cluster_state.go

```diff
@@ -225,15 +224,17 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			if ns, ok := c.suspendedNodes[update.Id]; ok {
 				if !ns.ready() {
 					// Adding a node that's already suspended as non-ready, leave it in that state until ready.
-					log.Infof("Suspended node re-added but still awaiting readiness check %v (%#v)", update.Id, ns)
+					log.Infof("Suspended node re-added but still awaiting readiness check %v (%s)", update.Id,
+						ns.node.String())
 				} else if ns.timeLost != nilTime {
 					// This node was suspended as lost earlier, we can recover it now.
 					ns.timeLost = nilTime
 					c.nodes[update.Id] = ns
 					delete(c.suspendedNodes, update.Id)
-					log.Infof("Recovered suspended node %v (%#v), %s", update.Id, ns, c.status())
+					log.Infof("Recovered suspended node %v (%s), %s", update.Id, ns.node.String(), c.status())
```

Comment:
It's not clear to me - why did you change from logging NodeState to logging Node?  We're losing a lot of useful information by doing this.

Reply:
fixed - was trying to get logging to print the wily node content instead of its address